src/__tests__/videoList.test.jsx

```diff
@@ -0,0 +1,45 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { act } from '@testing-library/react-hooks';
+import axios from '../config/axios';
+import VideoList from '../components/VideoList';
+import ThemeState from '../context/theme/themeState';
+import VideoState from '../context/video/videoState';
+
+import videosMock from '../mocks/youtube-videos-mock.json';
+
+jest.mock('../config/axios');
+describe('VideoList component tests', () => {
+  beforeEach(async () => {
+    axios.mockReturnValue(
+      Promise.resolve({
+        data: { ...videosMock },
+      })
+    );
```

Comment:
```js
axios.mockReturnValue(
      Promise.resolve({
        data: { ...videosMock },
      })
```

If this code is duplicated on another tests, you could figure it out how to reuse it